src/Common/src/Interop/Unix/libcorefx_io/Interop.libcorefx_io.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Why do we have both of these Interop.libcorefx_io.cs and Interop.GetFileStat.cs files?